glean_parser/lint.py

```diff
@@ -262,6 +262,36 @@ def check_expired_metric(
         yield ("Metric has expired. Please consider removing it.")
 
 
+def check_redundant_ping(
+    pings: Iterable[pings.Ping]
+) -> LintGenerator:
+    '''
+    Check if the pings contains 'ping' as the prefix or suffix, or 'ping' or 'custom'
+    '''
+    ping_words = sorted([_split_words(ping.name) for ping in pings])
```

Comment:
With only one ping being passed in this can become:
```suggestion
    ping_words = _split_words(ping.name)
```

Reply:
Got you!